tools/autograd/gen_python_functions.py

```diff
@@ -149,7 +149,8 @@
     'std::tuple<Tensor,Tensor,Tensor,int64_t>',
     'std::tuple<Tensor,Tensor,double,int64_t>',
     'std::vector<Tensor>',
-    'Scalar', 'bool', 'int64_t', 'void*', 'void'
+    'Scalar', 'bool', 'int64_t', 'void*', 'void',
+    'Quantizer*'
```

Comment:
do you need this?  I thought in the initial version we weren't going to expose the Quantizer to python.

Reply:
OK, will remove